templates/rag-gpt-crawler/rag_gpt_crawler/chain.py

```diff
@@ -0,0 +1,55 @@
+import json
+from pathlib import Path
+
+from langchain.chat_models import ChatOpenAI
+from langchain.embeddings import OpenAIEmbeddings
+from langchain.prompts import ChatPromptTemplate
+from langchain.pydantic_v1 import BaseModel
+from langchain.schema import Document
+from langchain.schema.output_parser import StrOutputParser
+from langchain.schema.runnable import RunnableParallel, RunnablePassthrough
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain.vectorstores import Chroma
+
+# Load output from gpt crawler
+path_to_gptcrawler = "/Users/rlm/Desktop/Code/gpt-crawler/output.json"
```

Comment:
@efriis this is hard-coded path to file produced by running the crawler. 

There may be a more elegant way to do this.